src/public/app/features/team/team-create/team-create.component.ts

```diff
@@ -1,17 +1,63 @@
-import { Component, OnInit } from '@angular/core';
+import { Component, OnChanges, OnInit } from '@angular/core';
+import { FormBuilder, FormControl, FormGroup, Validators } from '@angular/forms';
+import { ActivatedRoute, Router } from '@angular/router';
+import { switchMap } from 'rxjs/operators';
```

Comment:
Can you remove this import since it's not being used?

Reply:
Which import is unused?

Reply:
The switchMap